graph/api/src/main/java/org/jboss/windup/graph/service/ProjectService.java

```diff
@@ -1,15 +1,62 @@
 package org.jboss.windup.graph.service;
 
+import org.apache.commons.lang.RandomStringUtils;
 import org.jboss.windup.graph.GraphContext;
 import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.graph.model.WindupConfigurationModel;
+import org.jboss.windup.graph.model.resource.FileModel;
+import org.jboss.windup.util.PathUtil;
+
+import java.nio.file.Path;
 
 /**
  * Provides useful methods for querying, creating, and updating {@link ProjectModel} instances.
  */
 public class ProjectService extends GraphService<ProjectModel>
 {
+    public static final String SHARED_LIBS_APP_NAME = "<shared-libs>";
```

Comment:
That is possible if it becomes a problem. I'm not sure that it will make for a great user experience to add another configuration option here, though. Our current configuration options are daunting enough, IMO.